src/lib/country-data.js

```diff
@@ -26,3 +59,66 @@ module.exports.subdivisionOptions = Object.keys(countries).reduce((subByCountry,
         });
     return subByCountry;
 }, {});
+
+/**
+ * Take startingCountryOptions, and override any number of its country labels.
+ * E.g., to use the German and Italian names of Germany and Italy, do:
+ *     overrideLabels({de: 'Deutschland', it: 'Italia'})
+ * @param {array} startingCountryOptions array of country objects
+ *     like [{value: 'ca', label: 'Canada'}, ...]
+ * @param {object} countryNameOverrides map of country codes we should override,
+ *     to labels to use instead of default
+ * @returns {array} revised array of country objects, with overridden labels
+ */
+const overrideLabels = module.exports.overrideLabels = (startingCountryOptions, countryNameOverrides) => {
```

Comment:
I don't see this one used outside the module itself. Maybe it shouldn't be exported?

Reply:
it needs to be exported to be unit tested.